cpp/source/deal.II/electrochemical_physics.cc

```diff
@@ -0,0 +1,14 @@
+/* Copyright (c) 2016, the Cap authors.
+ *
+ * This file is subject to the Modified BSD License and may not be distributed
+ * without copyright and license information. Please refer to the file LICENSE
+ * for the text and further information on this license.
+ */
+
+#include <cap/deal.II/electrochemical_physics.templates.h>
```

Comment:
I don't really like the ``cap/deal.II`` and I'm pretty sure you don't need the symlink you added for this.
Copying the headers to the install prefix in the include directory will be a real pain...